File: src/com/l2kt/gameserver/network/serverpackets/RecipeShopMsg.java
package com.l2kt.gameserver.network.serverpackets;

import com.l2kt.gameserver.model.actor.instance.Player;

public class RecipeShopMsg extends L2GameServerPacket
{
	private final Player _activeChar;
	
	public RecipeShopMsg(Player player)
	{
		_activeChar = player;
	}
	
	@Override
	protected final void writeImpl()
	{
		writeC(0xdb);
		writeD(_activeChar.getObjectId());
		writeS(_activeChar.getCreateList().getStoreName());
	}
}

File: src/com/l2kt/gameserver/network/serverpackets/RecipeShopMsg.kt
package com.l2kt.gameserver.network.serverpackets

import com.l2kt.gameserver.model.actor.instance.Player

class RecipeShopMsg(private val _activeChar: Player) : L2GameServerPacket() {

    override fun writeImpl() {
        writeC(0xdb)
        writeD(_activeChar.objectId)
        writeS(_activeChar.createList.storeName)
    }
}